docs/docs/04-standard-library/01-cloud/api-reference.md

```diff
@@ -17,339 +17,178 @@ sidebar_position: 100
 
-#### Initializers <a name="Initializers" id="@winglang/sdk.cloud.Api.Initializer"></a>
+#### Initializer <a name="Initializer" id="@winglang/sdk.cloud.Api.Initializer"></a>
 
 ```wing
 bring cloud;
 
 new cloud.Api(props?: ApiProps)
 ```
 
-| **Name** | **Type** | **Description** |
+| **Parameter** | **Type** | **Description** |
 | --- | --- | --- |
-| <code><a href="#@winglang/sdk.cloud.Api.Initializer.parameter.props">props</a></code> | <code><a href="#@winglang/sdk.cloud.ApiProps">ApiProps</a></code> | *No description.* |
-
----
-
-##### `props`<sup>Optional</sup> <a name="props" id="@winglang/sdk.cloud.Api.Initializer.parameter.props"></a>
-
-- *Type:* <a href="#@winglang/sdk.cloud.ApiProps">ApiProps</a>
+| <code>props</code> | <code><a href="#@winglang/sdk.cloud.ApiProps">ApiProps</a></code> | *No description.* |
 
 ---
 
 #### Methods <a name="Methods" id="Methods"></a>
 
 | **Name** | **Description** |
 | --- | --- |
-| <code><a href="#@winglang/sdk.cloud.Api.connect">connect</a></code> | Add a inflight handler to the api for CONNECT requests on the given path. |
-| <code><a href="#@winglang/sdk.cloud.Api.delete">delete</a></code> | Add a inflight handler to the api for DELETE requests on the given path. |
-| <code><a href="#@winglang/sdk.cloud.Api.get">get</a></code> | Add a inflight handler to the api for GET requests on the given path. |
-| <code><a href="#@winglang/sdk.cloud.Api.head">head</a></code> | Add a inflight handler to the api for HEAD requests on the given path. |
-| <code><a href="#@winglang/sdk.cloud.Api.options">options</a></code> | Add a inflight handler to the api for OPTIONS requests on the given path. |
-| <code><a href="#@winglang/sdk.cloud.Api.patch">patch</a></code> | Add a inflight handler to the api for PATCH requests on the given path. |
-| <code><a href="#@winglang/sdk.cloud.Api.post">post</a></code> | Add a inflight handler to the api for POST requests on the given path. |
-| <code><a href="#@winglang/sdk.cloud.Api.put">put</a></code> | Add a inflight handler to the api for PUT requests on the given path. |
+| <code>connect</code> | Add a inflight handler to the api for CONNECT requests on the given path. |
+| <code>delete</code> | Add a inflight handler to the api for DELETE requests on the given path. |
+| <code>get</code> | Add a inflight handler to the api for GET requests on the given path. |
+| <code>head</code> | Add a inflight handler to the api for HEAD requests on the given path. |
+| <code>options</code> | Add a inflight handler to the api for OPTIONS requests on the given path. |
+| <code>patch</code> | Add a inflight handler to the api for PATCH requests on the given path. |
+| <code>post</code> | Add a inflight handler to the api for POST requests on the given path. |
+| <code>put</code> | Add a inflight handler to the api for PUT requests on the given path. |
 
 ---
 
-##### `connect` <a name="connect" id="@winglang/sdk.cloud.Api.connect"></a>
-
+**<code>connect</code>**
```

Comment:
Does this mean that it is now impossible to link to a specific method?

Can we leave the anchors?